### PR TITLE
Disable CGO support

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#7135

e2e
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/5345/

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -6,7 +6,7 @@ source $(dirname $0)/version
 LINKFLAGS="-X main.Version=$VERSION
            -X main.GitCommit=$GITCOMMIT
            -X main.BuildDate=$BUILDDATE
-           -linkmode external -extldflags -static"
+            -extldflags -static -s"
```
Comment (c3y1huang) [inline scripts/build:9]: The indent looks off.

Comment (c3y1huang) [reply scripts/build:9]: What is the `-s` for?

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: scripts/build (hunk 1 above)
Comment (derekbit) [reply scripts/build:9]: `Remove all symbol table and relocation information from the executable.` 
Same as https://github.com/longhorn/longhorn-manager/blob/master/scripts/build#L9

Ref: https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html

Comment (c3y1huang) [reply scripts/build]: I am curious how with or without the `-s` would impact the share-manager. Or are we adding this for consistency?

Comment (derekbit) [reply scripts/build]: Just for consistency.
It is like `strip` that removes the symbols which are usually for debugging tools such as gdb.

**Review by c3y1huang — APPROVED** [review]

**Pull request merged**